client/testing.go

```diff
@@ -23,14 +23,21 @@ const (
 	FakeResourceGroup  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/test/providers/Microsoft.Storage/storageAccounts/cqprovidertest"
 )
 
+func clearIgnoreInTests(t *schema.Table) {
```

Comment:
nit: is this an issue in all providers? i.e this needs to be part of the SDK?

Reply:
I think it is the same in all providers. IgnoreInTests is set to `false` only on top level table.

Not sure what needs to be moved to SDK. Just this little helper function?

Reply:
There is a flag already that does just that on the SDK side. Updated the commit accordingly.